pkg/subctl/cmd/gather/tally.go

```diff
@@ -0,0 +1,356 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package gather
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"path/filepath"
+
+	subctlversion "github.com/submariner-io/submariner-operator/pkg/version"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// Embed the file content as string.
+//go:embed layout.gohtml
+var layout string
+/*
```

Comment:
embed does not work for me locally so checking if it works in CI and hence commented this var here. Final patch will be with embed (if it works) or defining a template like this in a string.